presto-openapi/src/main/resources/query.yaml

```diff
@@ -0,0 +1,122 @@
+openapi: 3.0.0
+info:
+  title: Presto Query Info
+  description: Optional multiline or single-line description in [CommonMark](http://commonmark.org/help/) or HTML.
+  version: 1
+servers:
+  - url: http://api.example.com/v1
+    description: Optional server description, e.g. Main (production) server
+  - url: http://staging-api.example.com
+    description: Optional server description, e.g. Internal staging server for testing
```

Comment:
Let's use `http://localhost:8080` here for the URL as that's the endpoint for the API when running Presto locally. The description should also be updated to reflect that. Something like: `Presto endpoint when running locally`